include/LLGL/Surface.h

```diff
@@ -88,6 +88,15 @@ class LLGL_EXPORT Surface : public Interface
         LLGL_DEPRECATED("Surface::ResetPixelFormat is deprecated since 0.04b; No need to reset pixel format anymore!")
         virtual void ResetPixelFormat();
 
+        /**
+        \brief Is the surface is using Wayland
+        \remarks Only works on Linux
+        \return True if the surface is using Wayland
+        */
+        virtual bool IsWayland() const {
```

Comment:
I think this new interface function is redundant if we already have this information through `NativeHandle::type`. You just call `GetNativeHandle()` and this field tells you what the surface type is.